app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_ls = command.split()
```

Comment:
please, dont use short forms in variable names. It's better to use whole words to make it more descriptive